.env.sample

```diff
@@ -1,21 +1,21 @@
 BITCOIND_URL=
-EMILY_URL=
-MEMPOOL_API_URL=https://mempool.space/api
-PUBLIC_MEMPOOL_URL=https://mempool.space
+EMILY_URL=http://temp.sbtc-emily-dev.com
+MEMPOOL_API_URL=https://mempool.bitcoin.regtest.hiro.so/api
+PUBLIC_MEMPOOL_URL=https://mempool.bitcoin.regtest.hiro.so
```

Comment:
Can we use devenv instead (see https://github.com/stacks-sbtc/sbtc/blob/main/docker/docker-compose.yml#L248)? I don't think we want local runs to interact with testnet deployment.
Then we can add a paragraph on the readme about this (eg, the default env runs against sbtc devenv, go to this repo for the instructions).

Reply:
yes wanted to do this initially makes sense

Reply:
We recently switched to explicit ipv4 binding for devenv, can we replace `localhost` with `127.0.0.1` here as well?